### ISSUE TITLE
1.20.1 Disinger armor

### ISSUE THREAD

**Issue by dinobossytnew** [PluginBugs/Issues-ItemsAdder#2826]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

dinobossyt

### Describe the solution you'd like

Hello, I was thinking that I have a lot of armor, and I'm tired of keeping them in leather. Then I thought, why not ask Lone if he knows any way to store them in metal or other materials in version 1.20.1?
I thought the best way would be for Lone to implement the armor design system, allowing the use of different layers. I'm not sure if this is possible, but it would be great if we could find a solution.

### Is your feature request related to a problem?

Yes basically a lot of custom leather armors that we make sometimes do not receive the armor effects correctly configured in these cases we report bugs but being many things that are reported or left for the last to add or as low priority since it is not something that affects the plugin a lot there are always alternatives to add effects or attributes to the armors.

### Describe alternatives you've considered

The alternatives are to use customsitems or executableitems you could also use mmoitems but it would be more than a waste to add attributes to the armor, remember that the effect of this does not change, it could be good but it is not what we are looking for.

### Additional context

Basically, we are wondering if it is possible in version 1.20.1 to introduce support for armor designs that function like layers and allow customization. We understand that there may be a limit to the number of custom armor sets we can create, let's say around 50 or 40 sets. However, even with this limitation, it would provide us with a more advanced way to customize diamond armor.

**Comment by LoneDev6** [comment]
> any way to store them in metal or other materials in version 1.20.1?

No sadly, not that I'm aware of.

> Yes basically a lot of custom leather armors that we make sometimes do not receive the armor effects correctly configured 

No idea what you mean, if you have found any bug report it on Github.
